src/test/java/com/connexta/search/index/IndexTests.java

```diff
@@ -288,4 +326,27 @@ private static SolrInputDocument hasIndexFieldValues(
                 && StringUtils.equals(
                     (String) solrInputDocument.getField("contents").getValue(), contents));
   }
+
+  @ParameterizedTest(name = "400 Bad Request if ID is {0}")
+  @ValueSource(strings = {"1234567890123456789012345678901234", "+0067360b70e4acfab561fe593ad3f7a"})
+  void testBadRequests(String id) throws Exception {
```

Comment:
✏️ `testsBadDatasetIds` because there are other bad requests that aren't included in these params